Sources/HaCTML/Attributes.swift

```diff
@@ -1,13 +1,22 @@
 private func stringAttribute(_ key: String) -> AttributeKey<String> {
-  return .init(key, apply: { .text($0) })
+  return AttributeKey(key, apply: { .text($0) })
```

Comment:
I kept stumbling on trying to read this `.init`

Reply:
Good call, should we give the other attribute functions the same treatment?

Reply:
I spoke to soon, you did exactly that!